.gitignore

```diff
@@ -377,4 +377,4 @@ android/pytorch_android_torchvision/.cxx
 .pyre_configuration.codenav
 .arcconfig
 .stable_pyre_client
-.pyre_client
+.pyre_client
```

Comment:
Please put the newline back

Reply:
Done.

Reply:
Sorry, did you mean adding a newline after line 382 ?

Reply:
![image](https://github.com/user-attachments/assets/24d74c75-c394-42a6-914b-dfd4d795bce9)

This is what I'm seeing